.github/workflows/appengine-deploy.yml

```diff
@@ -6,9 +6,13 @@ on:
       - main 
   pull_request:
 
+# The golden hash represents the commit of the baseline images that we want to compare our current app against. 
+# We will need to update the hard-coded hash to be the hash of the most recent commit to main.
```

Comment:
Please reflow to 80-characters.

I think we only need to change the hash if there are visible changes.